lib/hapi.js

```diff
@@ -19,13 +28,10 @@ module.exports = function (config) {
       options: {config: config}
     }
   })/*, [{
-    register: require('hoodie-server-store'),
-    options: _.defaults({prefix: '/hoodie/store'}, defaultOpts)
-  }, {
-    register: require('hoodie-server-account/routes/session'),
+    register: require('@hoodie/server-account-node-sessions'),
```

Comment:
It’s out of scope for this PR, but the new options required for `hoodie-server-account` are `options.db` and `options.admins` as described here: https://github.com/hoodiehq/hoodie-server-account/tree/node-sessions/plugin#optionsdb

Does that make sense from your perspective? We use the `options.admins` map to see if someone tries to sign in as an admin user, as admins don’t necessarly have a `_users` doc